### PR TITLE
Issue MapViz no map viewed

### PR DESCRIPTION
**Pull request by mattiadutto**
I tried using the code of the [Navigating Using GPS Localization](https://navigation.ros.org/tutorials/docs/navigation2_with_gps.html) and unfortunately, out of the box, it doesn't work (of course I added the bing-api key). I'm not able to see a bing map on the MapViz window. Searching on the web for a solution I saw that by replacing the MapViz configuration with this one (taken [here](https://github.com/swri-robotics/mapviz/issues/575#issuecomment-375669263)) it works. 

System configuration: Ubuntu 22.04.4 LTS + ROS2 Humble + Navigation2 1.1.13

### CODE REVIEW

File: nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml [hunk 1]
```diff
@@ -6,18 +6,13 @@
 # - global costmap is set to be rolling to allow the robot to traverse big environment by
 #   following successive relatively close waypoints that fit in a smaller rolling costmap
 
-bt_navigator:
+t_navigator:
```
Comment (SteveMacenski) [inline nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml:9]: Typo

File: nav2_gps_waypoint_follower_demo/config/gps_wpf_demo.mvc [hunk 2]
```diff
@@ -7,14 +7,13 @@ enable_antialiasing: true
 show_displays: true
 show_status_bar: true
 show_capture_tools: true
-window_width: 1920
-window_height: 1016
-view_scale: 0.564473748
-offset_x: -55.0469894
-offset_y: -4.65194941
+window_width: 1000
```
Comment (SteveMacenski) [inline nav2_gps_waypoint_follower_demo/config/gps_wpf_demo.mvc:10]: Why does changing these values fix the problem?

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

File: nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml [hunk 3]
```diff
@@ -74,9 +65,6 @@ bt_navigator:
       - nav2_assisted_teleop_cancel_bt_node
       - nav2_drive_on_heading_cancel_bt_node
       - nav2_is_battery_charging_condition_bt_node
-    error_code_names:
```
Comment (SteveMacenski) [inline nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml:77]: Readd. These are for iron, rolling, jazzy and newer. Documentation targets the latest distributions.

Comment (mattiadutto) [reply nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml:77]: Sorry I did the pull request just for the first commit and commit other changes for working with Humble. This file and the nav2_map_parameters where not intended to be added to the PR.

File: nav2_gps_waypoint_follower_demo/config/gps_wpf_demo.mvc (hunk 2 above)
Comment (mattiadutto) [reply nav2_gps_waypoint_follower_demo/config/gps_wpf_demo.mvc:10]: I think that what is making the system working under Humble is the remove of this line of code `image_transport: raw` I didn't try any possible edits.

File: nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml (hunk 3 above)
Comment (SteveMacenski) [reply nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml:77]: Please update and I'll take a look again :-)

File: nav2_gps_waypoint_follower_demo/config/gps_wpf_demo.mvc (hunk 2 above)
Comment (SteveMacenski) [reply nav2_gps_waypoint_follower_demo/config/gps_wpf_demo.mvc:10]: OK, lets just change that item then :-)

**Comment by SteveMacenski** [comment]
Adding a new humble branch shortly with the changes that fix the issue without the typos

**Pull request closed without merging**